spec/models/polling/scheduler_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:scheduler) { Polling::Scheduler.new }
 
   after :each do
-    scheduler.rufus_scheduler.shutdown
+    #scheduler.rufus_scheduler.shutdown
```

Comment:
have you meant to leave the comment here?

Reply:
Thanks